package/network/services/ead/src/tinysrp/t_getpass.c

```diff
@@ -44,7 +44,7 @@ static  struct  sigaction sigact;
 #endif
 
 /*ARGSUSED*/
-static RETSIGTYPE
+void
```

Comment:
why remove static?

Reply:
Ah, that was just my tired brain. I've changed it, thanks for checking.